regolith/mongoclient.py

```diff
@@ -102,6 +105,27 @@ def import_yamls(dbpath: str, dbname: str, host: str = None, uri: str = None) ->
     return
 
 
+def load_mongo_col(col: Collection) -> dict:
```

Comment:
is there a way to write a test for this?

Reply:
Yes. I will work on it.